File: app/src/main/java/com/simplemobiletools/calculator/operation/MinusOperation.java
package com.simplemobiletools.calculator.operation;

import com.simplemobiletools.calculator.operation.base.BinaryOperation;
import com.simplemobiletools.calculator.operation.base.Operation;

public class MinusOperation extends BinaryOperation implements Operation {
    protected MinusOperation(double baseValue, double secondValue) {
        super(baseValue, secondValue);
    }

    @Override
    public double getResult() {
        return baseValue - secondValue;
    }
}

File: app/src/main/kotlin/com/simplemobiletools/calculator/operation/MinusOperation.kt
package com.simplemobiletools.calculator.operation

import com.simplemobiletools.calculator.operation.base.BinaryOperation
import com.simplemobiletools.calculator.operation.base.Operation

class MinusOperation(baseValue: Double, secondValue: Double) : BinaryOperation(baseValue, secondValue), Operation {

    override fun getResult() = baseValue - secondValue
}
